modules/docs/lib/widgetUtils.tsx

```diff
@@ -102,31 +157,69 @@ export const SymbolDialog = ({value}: SymbolDialogProps) => {
         </Dialog.Target>
+
         {renderTypeParameters(value.typeParameters)}
-      </>
-      <Dialog.Popper>
-        <Dialog.Card maxHeight="50vh" maxWidth="90vh">
-          <Dialog.CloseIcon />
-          <Dialog.Heading>{value.name}</Dialog.Heading>
-          <Dialog.Body>
-            <RenderContext.Provider value="table">
-              <IndentLevelContext.Provider value={0}>
-                {symbol ? (
-                  <SymbolDoc name={value.name} headingStart={3} hideHeading />
-                ) : (
-                  <>
-                    <p>Basic type information:</p>
-                    <pre>
-                      <code>{value.value}</code>
-                    </pre>
-                  </>
-                )}
-              </IndentLevelContext.Provider>
-            </RenderContext.Provider>
-          </Dialog.Body>
-        </Dialog.Card>
-      </Dialog.Popper>
-    </Dialog>
+
+        <Dialog.Popper>
+          <Dialog.Card maxHeight="50vh" maxWidth="90vh">
+            <Dialog.CloseIcon />
+
+            <Dialog.Heading>
+              {value.name}{' '}
+              {breadcrumbsList.length >= 1 ? (
+                <Breadcrumbs aria-label="Breadcrumbs">
+                  <Breadcrumbs.List>
+                    {breadcrumbsList.map((item, index) => {
+                      return (
+                        <>
+                          {item === breadcrumbsList[breadcrumbsList.length - 1] ? (
+                            <Breadcrumbs.CurrentItem key={index}>{item}</Breadcrumbs.CurrentItem>
+                          ) : (
+                            <Breadcrumbs.Item key={index}>
+                              <Breadcrumbs.Link
+                                onClick={e => handleBreadcrumbClick(e, index)}
+                                href={'#'}
+                              >
+                                {item}
+                              </Breadcrumbs.Link>
+                            </Breadcrumbs.Item>
+                          )}
+                        </>
+                      );
+                    })}
+                  </Breadcrumbs.List>
+                </Breadcrumbs>
+              ) : null}
+            </Dialog.Heading>
+
+            <Dialog.Body>
+              <RenderContext.Provider value="table">
+                <IndentLevelContext.Provider value={0}>
+                  {symbol ? (
+                    <SymbolDoc
+                      name={
+                        breadcrumbsList.length >= 1
+                          ? breadcrumbsList[breadcrumbsList.length - 1]
+                          : value.name
+                      }
+                      headingStart={3}
+                      hideHeading
+                    />
+                  ) : (
+                    <>
+                      <p>Basic type information:</p>
+                      <pre>
+                        <code>{value.value}</code>
+                      </pre>
+                    </>
```

Comment:
Small thing, but this `Basic type information` fallback doesn't seem to be rendering any more. Try clicking the `Secondary Button` link in the ActionBar.Item props and then the `CanvasSystemIcon` type for the `icon` prop:

<img width="956" alt="Screenshot 2023-03-08 at 6 20 21 PM" src="https://user-images.githubusercontent.com/146020/223899227-b9be31b7-bf6e-43ab-bdf5-863b19cc2570.png">

This is what it looked like before:

<img width="1377" alt="Screenshot 2023-03-08 at 6 23 30 PM" src="https://user-images.githubusercontent.com/146020/223899525-ef2b86ee-7e31-42fd-b4e5-6ba6c46961a1.png">

Granted, `Basic type information` doesn't add much information, but I think it's still better than `Not found` which gives the impression something is broken.